packages/backend/src/managers/playground.ts

```diff
@@ -204,6 +207,26 @@ export class PlayGroundManager {
       }
     });
 
+    let contacted = false;
```

Comment:
Maybe we should create an utility function _somewhere_ that would manage the `retry` logics, we would reuse it later for other usage, with some timeout limits. See https://stackoverflow.com/a/45445291 for some retryable utility functions. I would prefer this approche as we could test each component individually (the retryable, the timeout, and finally the health state would be trivial)

Moreover, fetch is pretty bad at providing some custom timeout, we might not want wait for the maximum configured, see https://stackoverflow.com/questions/46946380/fetch-api-request-timeout

Reply:
Delayed to a follow up PR